src/core/logic.py

```diff
@@ -111,6 +111,8 @@ def resize_and_crop(img_path, size, crop_type='middle'):
         img = img.crop(box)
     else:
         img = img.resize((size[0], size[1]), Image.ANTIALIAS)
+        if img.mode == "CMYK":
```

Comment:
Should this be indented under the else here?

Reply:
indeed